recipes-kernel/linux/files/bcm2712-raspberrypi5-xen-scmi.dtso

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
considering that this dtso is for debug purposes and should be changed in future. Also it will definitely be changed in AOS, I think this should be moved to meta-xt-prod-devel-rpi5.
This approach will save us from need to change overlay here during migration to aos

Reply:
Hm. It's added to auto-generated u-boot script. How it can be solved?

Reply:
ok. updated - The prod layers can replace SCMI DT using XEN_DT_SCMI parameter.
This layer contains non-Xen specific overlay.

Reply:
My point was to add bbappend for u-boot-scr to meta-xt-prod-devel-rpi5. From my standpoint - only reference dtbs can be added here.